mslib/mscolab/server.py

```diff
@@ -259,7 +259,7 @@ def user_register_handler():
     try:
         if result["success"]:
             status_code = 201
-            if mscolab_settings.USER_VERIFICATION:
+            if mscolab_settings.MAIL_ENABLED:
                 status_code = 204
             token = generate_confirmation_token(email)
```

Comment:
from token to sendmail needs to be intended. This is only needed when MAIL_ENABLED is True

on the subject below please use the same prefix as for the other mail.

Reply:
That works all now as expected, only the MSColab Mail prefix is missing.